packages/react-components/react-tags/stories/src/InteractionTag/InteractionTagSelected.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { InteractionTag, InteractionTagPrimary, InteractionTagSecondary, makeStyles } from '@fluentui/react-components';
+import { CalendarMonthRegular, CalendarMonthFilled, bundleIcon } from '@fluentui/react-icons';
+
+const CalendarMonth = bundleIcon(CalendarMonthFilled, CalendarMonthRegular);
+
+const useContainerStyles = makeStyles({
+  container: {
+    columnGap: '10px',
+    display: 'flex',
+  },
+});
```

Comment:
_nit:_ Since these are going to a `div` they can be a `makeResetStyles` call instead.

Reply:
fixed